### DISCUSSION TITLE
Worry that with the update of the software, the current data will be incompatible in the future

### DISCUSSION THREAD

**Discussion by Angel0726** [AppFlowy-IO/AppFlowy#453] [General help wanted] [answered]
Worry that with the update of the software, the current data will be incompatible in the future

**Comment by annieappflowy** [comment, marked as answer]
Hi @Angel0726 , thanks for asking. We keep appflowy compatible with lower general available versions. That said, betas are not stable or fully tested. For example v0.0.4 (no suffix) will be heavily tested on its compatibility whereas v0.0.4-beta.2 is not.

**Reply by Angel0726** [reply]
can import data from notion

**Reply by lawweiliang** [reply]
> can import data from notion

Now no. 
Take a look on this yeah. 
https://github.com/AppFlowy-IO/AppFlowy/issues/20
